### PR TITLE
Style ux event page

### PR DESCRIPTION
**Pull request by Prerna290**
## Type
- ( ) Fix
- ( ) Refactor
- ( ) Style
- ( ) Docs
- (X) Feat
- ( ) Hotfix

### CODE REVIEW

File: NX
Comment (ArshdeepGrover) [inline NX:1]: delete this file

File: apps/commudle-admin/src/app/feature-modules/community-control-panel/components/community-events-list/community-events-list-public-page/community-events-list-public-page.component.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<div>
+<div class="main-div">
```
Comment (ArshdeepGrover) [inline apps/commudle-admin/src/app/feature-modules/community-control-panel/components/community-events-list/community-events-list-public-page/community-events-list-public-page.component.html]: try to use better class names

File: apps/commudle-admin/src/app/feature-modules/community-control-panel/components/community-events-list/community-events-list-public-page/community-events-list-public-page.component.scss [hunk 2]
```diff
@@ -1,3 +1,6 @@
-a {
-  margin: 0.5rem auto;
+.main-div {
+  @apply com-flex com-flex-wrap;
+  a {
+    margin: 0.5rem auto;
+  }
 }
```
Comment (ArshdeepGrover) [inline apps/commudle-admin/src/app/feature-modules/community-control-panel/components/community-events-list/community-events-list-public-page/community-events-list-public-page.component.scss]: try to use better class names
and use tailwind classes

File: apps/commudle-admin/src/app/feature-modules/data-forms/components/edit-data-form/edit-data-form.component.html [hunk 3]
```diff
@@ -237,19 +237,17 @@ <h3>Edit Form</h3>
...
-                        nbButton
-                        [nbContextMenu]="items"
-                        nbContextMenuTrigger="click"
-                        class="context-menu"
-                      >
-                        <nb-icon icon="more-vertical-outline" class="com-mb-8"></nb-icon>
-                      </button>
-                    </div>
-                  </nb-layout-column>
+                  <div class="layout-div">
```
Comment (ArshdeepGrover) [inline apps/commudle-admin/src/app/feature-modules/data-forms/components/edit-data-form/edit-data-form.component.html]: try to use better class names

File: apps/commudle-admin/src/app/feature-modules/data-forms/components/edit-data-form/edit-data-form.component.scss [hunk 4]
```diff
@@ -139,16 +139,14 @@
...
-      nb-layout-column {
-        div {
-          @apply com-flex com-justify-end;
-          height: 100%;
-          align-items: flex-end;
-        }
+      .layout-div {
+        @apply com-flex com-justify-end;
+        height: 100%;
+        align-items: flex-end;
```
Comment (ArshdeepGrover) [inline apps/commudle-admin/src/app/feature-modules/data-forms/components/edit-data-form/edit-data-form.component.scss]: try to use better class names

File: apps/commudle-admin/src/app/feature-modules/data-forms/components/edit-data-form/edit-data-form.component.scss [hunk 5]
```diff
@@ -158,7 +156,7 @@
           display: block;
         }
 
-        nb-layout-column {
+        .layout-div {
           display: block;
         }
```
Comment (ArshdeepGrover) [inline apps/commudle-admin/src/app/feature-modules/data-forms/components/edit-data-form/edit-data-form.component.scss:161]: Use tailwind

File: apps/commudle-admin/src/app/feature-modules/notifications/components/community-notifications/community-notifications.component.html [hunk 6]
```diff
@@ -1,5 +1,5 @@
-<button nbButton size="small" (click)="unread()" status="primary">Unread</button>
-<button nbButton size="small" (click)="allNotifications()" status="primary">All</button>
+<button nbButton size="small" (click)="unread()" status="primary" class="unread">Unread</button>
+<button nbButton size="small" (click)="allNotifications()" status="primary" class="allButton">All</button>
```
Comment (ArshdeepGrover) [inline apps/commudle-admin/src/app/feature-modules/notifications/components/community-notifications/community-notifications.component.html]: classes name should be all-notifications 
unread-notifications

**Review by ArshdeepGrover — CHANGES_REQUESTED** [review]

File: NX
Comment (Prerna290) [reply NX:1]: Done

File: apps/commudle-admin/src/app/feature-modules/community-control-panel/components/community-events-list/community-events-list-public-page/community-events-list-public-page.component.html (hunk 1 above)
Comment (Prerna290) [reply apps/commudle-admin/src/app/feature-modules/community-control-panel/components/community-events-list/community-events-list-public-page/community-events-list-public-page.component.html]: Changed

File: apps/commudle-admin/src/app/feature-modules/community-control-panel/components/community-events-list/community-events-list-public-page/community-events-list-public-page.component.scss (hunk 2 above)
Comment (Prerna290) [reply apps/commudle-admin/src/app/feature-modules/community-control-panel/components/community-events-list/community-events-list-public-page/community-events-list-public-page.component.scss]: Done

File: apps/commudle-admin/src/app/feature-modules/data-forms/components/edit-data-form/edit-data-form.component.html (hunk 3 above)
Comment (Prerna290) [reply apps/commudle-admin/src/app/feature-modules/data-forms/components/edit-data-form/edit-data-form.component.html]: Done

File: apps/commudle-admin/src/app/feature-modules/data-forms/components/edit-data-form/edit-data-form.component.scss (hunk 4 above)
Comment (Prerna290) [reply apps/commudle-admin/src/app/feature-modules/data-forms/components/edit-data-form/edit-data-form.component.scss]: Done

File: apps/commudle-admin/src/app/feature-modules/data-forms/components/edit-data-form/edit-data-form.component.scss (hunk 5 above)
Comment (Prerna290) [reply apps/commudle-admin/src/app/feature-modules/data-forms/components/edit-data-form/edit-data-form.component.scss:161]: Done

File: apps/commudle-admin/src/app/feature-modules/notifications/components/community-notifications/community-notifications.component.html (hunk 6 above)
Comment (Prerna290) [reply apps/commudle-admin/src/app/feature-modules/notifications/components/community-notifications/community-notifications.component.html]: Done

**Pull request merged**